src/components/FormBook.js

```diff
@@ -1,13 +1,15 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Kindly remove this eslint disable line and add prop types in the component.